allure-selenide/src/main/java/io/qameta/allure/selenide/AllureSelenide.java

```diff
@@ -68,6 +81,15 @@ public AllureSelenide savePageSource(final boolean savePageHtml) {
         return this;
     }
 
+    public AllureSelenide saveLogsFrom(String logType) {
```

Comment:
Can we create an enum for log types? That way it would be much easier for users to use it

Reply:
Suggested API: 

```
AllureSelenide enableLogs(LogType type, LogLevel level);
AllureSelenide disableLogs(LogType type);
```